dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -1195,14 +1201,19 @@ public User setUserPrivilege(@Parameter(hidden = true, name = "user")@Auth User
             + "", description = "A list of GitHub Events for the logged in user", content = @Content(array = @ArraySchema(schema = @Schema(implementation = LambdaEvent.class))))
     @ApiOperation(value = "See OpenApi for details")
     public List<LambdaEvent> getUserGitHubEvents(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User authUser,
-            @Min(0) @QueryParam("offset") Integer offset,
-            @DefaultValue(PAGINATION_LIMIT) @QueryParam("limit") Integer limit,
+            @Parameter(description = PAGINATION_OFFSET_TEXT) @Min(0) @QueryParam("offset") Integer offset,
+            @Parameter(description = PAGINATION_LIMIT_TEXT) @Max(MAX_PAGINATION_LIMIT) @DefaultValue(PAGINATION_LIMIT) @QueryParam("limit") Integer limit,
             @DefaultValue("") @QueryParam("filter") String filter,
             @DefaultValue("dbCreateDate") @QueryParam("sortCol") String sortCol,
-            @DefaultValue("desc") @QueryParam("sortOrder") String sortOrder) {
+            @DefaultValue("desc") @QueryParam("sortOrder") String sortOrder,
+            @Context HttpServletResponse response) {
+        int adjustedLimit = (int) Math.min(MAX_PAGINATION_LIMIT, limit);
```

Comment:
Do we need to do the `adjustedLimit`? I thought `@Max` handles the validation and throws an error if the value exceeds the max

Reply:
Good point, copied this from elsewhere, so going to make this more consistent